CanHazFunny/CanHazFunny/Jester.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+
+namespace CanHazFunny;
+
+public class Jester(IOutput jokeOutput, IJokeService jokeService)
+{
+    public IOutput JokeOutput { get; } = jokeOutput ?? throw new ArgumentNullException(nameof(jokeOutput));
+    public IJokeService JokeService { get; } = jokeService ?? throw new ArgumentNullException(nameof(jokeService));
+    public string BannedSubstring { get; set; } = "chuck norris";
```

Comment:
I would make BannedSubstring readonly since it doesn't look like it would change ever.
```suggestion
    public string BannedSubstring { get; } = "chuck norris";
```